Annotato/Annotato/Persistence/LocalPersistenceManager/AnnotatoLocal.xcdatamodeld/AnnotatoLocal.xcdatamodel/contents

```diff
@@ -43,21 +44,23 @@
         <attribute name="ownerId" attributeType="String"/>
         <attribute name="updatedAt" optional="YES" attributeType="Date" usesScalarValueType="NO"/>
         <relationship name="annotationEntities" optional="YES" toMany="YES" deletionRule="Cascade" destinationEntity="AnnotationEntity" inverseName="documentEntity" inverseEntity="AnnotationEntity"/>
-        <relationship name="documentShareEntities" optional="YES" toMany="YES" deletionRule="Cascade" destinationEntity="DocumentShareEntity" inverseName="documentEntity" inverseEntity="DocumentShareEntity"/>
     </entity>
-    <entity name="DocumentShareEntity" representedClassName="DocumentShareEntity" syncable="YES">
+    <entity name="SelectionBoxEntity" representedClassName="SelectionBoxEntity" syncable="YES">
         <attribute name="createdAt" optional="YES" attributeType="Date" usesScalarValueType="NO"/>
         <attribute name="deletedAt" optional="YES" attributeType="Date" usesScalarValueType="NO"/>
+        <attribute name="endPointX" attributeType="Double" defaultValueString="0.0" usesScalarValueType="YES"/>
+        <attribute name="endPointY" attributeType="Double" defaultValueString="0.0" usesScalarValueType="YES"/>
         <attribute name="id" attributeType="UUID" usesScalarValueType="NO"/>
-        <attribute name="recipientId" attributeType="String"/>
+        <attribute name="startPointX" attributeType="Double" defaultValueString="0.0" usesScalarValueType="YES"/>
+        <attribute name="startPointY" attributeType="Double" defaultValueString="0.0" usesScalarValueType="YES"/>
         <attribute name="updatedAt" optional="YES" attributeType="Date" usesScalarValueType="NO"/>
-        <relationship name="documentEntity" optional="YES" maxCount="1" deletionRule="Nullify" destinationEntity="DocumentEntity" inverseName="documentShareEntities" inverseEntity="DocumentEntity"/>
+        <relationship name="annotationEntity" optional="YES" maxCount="1" deletionRule="Nullify" destinationEntity="AnnotationEntity" inverseName="selectionBoxEntity" inverseEntity="AnnotationEntity"/>
```

Comment:
I think we may need to remove this in future if we choose to have selection boxes that can belong to different entities? But okay for now

Reply:
In core data I think that would just be another relationship.